RSS Reader/RSS Reader/Categories/NSError/NSError+UserFriendlyError.m

```diff
@@ -0,0 +1,37 @@
+//
+//  NSError+UserFriendlyError.m
+//  RSS Reader
+//
+//  Created by Lina Loyko on 12/7/20.
+//
+
+#import "NSError+UserFriendlyError.h"
+
+
+static NSString * const kParserErrorTitle = @"Parse error";
```

Comment:
I would better create a localized file and put error strings there.

Reply:
Done